### PR TITLE
Remove Internet Explorer from browser detection

### PR DESCRIPTION
**Pull request by jonkoops**
Removes Internet Explorer from the browser detection code and removes all related code based on these checks. This removes the following APIs:

- `L.Browser.ie`
- `L.Browser.ielt9`
- `L.Browser.ie3d`

### CODE REVIEW

**Review by IvanSanchez — APPROVED** [review]
This feels like an underwhelming historic moment.

I can't see anything wrong with the code changes.

**Review by Malvoz — APPROVED** [review]

**Review by Falke-Design — APPROVED** [review]
Nice!
It's important that we make in the changelog clearly visible all API changes, but I think this is clear

**Comment by jonkoops** [comment]
@Falke-Design Yeah, I was thinking we could perhaps already create a draft release for 2.0 with all the breaking changes and migration guide. That way we don't have to retroactively compile it all.

**Comment by Falke-Design** [comment]
Sounds good. But let us only add for now the main things like breaking / API changes so that we don't forget them. And then before the release, I go over all the commits again and add everything else

**Comment by jonkoops** [comment]
Exactly what I was thinking, I'll get on that after merging this in.

File: docs/examples/choropleth/example.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ css: "#map {
 			fillOpacity: 0.7
 		});
 
-		if (!L.Browser.ie && !L.Browser.opera && !L.Browser.edge) {
+		if (!L.Browser.opera && !L.Browser.edge) {
```
Comment (mourner) [inline docs/examples/choropleth/example.md:95]: BTW, the other two switches here are suspicious. For `edge`, is this detecting the old IE-based Edge, not the Chromium one? And why would Opera not work here if it's also pretty much a flavour of Chrome now? Maybe we could get rid of both of these too.

**Pull request merged**